docs/_index.md

```diff
@@ -223,7 +222,13 @@ public class App {
 {{% choosable language yaml %}}
 
 ```yaml
-# Coming soon
+name: Pulumi Program
+runtime: yaml
+resources:
+  pet:
+    type: random:RandomPet
+    properties:
+      length: 2
```

Comment:
FWIW `length` defaults to `2` so I assume this isn't required (but perhaps still useful in the example for YAML).

No context was given why it differed from the other choices which called a `Pet` function with `hi` / `pet` value as the unique resource name (_which I assume is the `resources.pet` here, while `type: random:RandomPet` maps directly to the `RandomPet()` method of the generated SDK, unlike the other examples that somehow don't map to the SDK method and instead like for JS with `new random.Pet("hi");`_ 🤷‍♂️ )

Refs:
- https://www.pulumi.com/registry/packages/terraform-provider/
- https://www.pulumi.com/registry/packages/random/api-docs/randompet/
- https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/pet
- https://github.com/pulumi/pulumi-random/blob/v4.16.7/sdk/nodejs/randomPet.ts#L86

As someone who's never used Pulumi or Terraform but is familiar with the language support, the inconsistency between examples isn't helpful since AFAIK this is two layers of abstraction to grok as a reference to adapt to a real-world configuration.

Reply:
I think the way that properties are passed in YAML is a bit less intuitive than in other languages, hence why I added a length parameter there. I think it helps to illustrate how one would pass other parameters.

In general, I wouldn't expect most users to look at the code examples for multiple languages - I'd expect users tend to use one language for pulumi and they would mostly look at that language for examples.